app/services/sfn_pipeline_service.rb

```diff
@@ -0,0 +1,133 @@
+class SfnPipelineService
+  # This service is responsible for dispatching a request to the
+  # StepFunctions-based pipeline.
+  # It generates jsons for all the pipeline run stages, converts them to WDL,
+  # creates the Step Function's input JSON and start SFN execution
```

Comment:
Would you mind changing the name to `SfnPipelineDispatchService` or something similar? I can imagine creating more services in the future that interact with the sfn pipeline - I might have to for the Pipeline Viz

Reply:
Done.